### PR TITLE
Closes Print Stylesheet #69 | @media print styles added.

### PR DESCRIPTION
**Pull request by jdromero88**
Closes print Stylesheet #69

### CODE REVIEW

File: wp-content/themes/reconasia/assets/_scss/style.scss [hunk 1]
```diff
@@ -13,3 +13,4 @@
 @use 'components/share';
 @use 'layout/header';
 @use 'layout/footer';
+@use 'modules/print';
```
Comment (jnschrag) [inline wp-content/themes/reconasia/assets/_scss/style.scss:16]: We don't want to include the print styles as a part of our global styles. The idea is by separating them out into their own stylesheet, which is separately loaded in via `functions.php` is that these styles will only ever be loaded in if the user is printing the site - we don't want to also load them just during regular usage of the site.

File: wp-content/themes/reconasia/assets/_scss/print.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+@use 'modules/print';
```
Comment (jnschrag) [inline wp-content/themes/reconasia/assets/_scss/print.scss:1]: I don't think you need to use `@use` here - just copy the contents of `/modules/_print.scss` directly into `/_scss/print.scss`.

**Review by jnschrag — CHANGES_REQUESTED** [review]
Thanks for taking a first pass at this! I left some inline comments, but some other things to note:

- Hide the social share
- The page header is missing, which means the post title, author information, etc. is all missing too
- Let's remove the max-width of 680px on the content for print, so we're using the space on the printed paper effectively
- Let's also set the font size of the body content to 16px for print (it'll still be big enough to read, but won't take up as much space = less pages to print)

Also, as a heads up, you can use your [dev tools](https://developer.chrome.com/docs/devtools/css/print-preview/) to force print preview mode for sites. Not sure if you knew this already, but wanted to share!

File: wp-content/themes/reconasia/assets/_scss/style.scss (hunk 1 above)
Comment (jdromero88) [reply wp-content/themes/reconasia/assets/_scss/style.scss:16]: Thats makes sense! thanks for the clarification!

**Comment by jdromero88** [comment]
Hi Jacque, I added max widht to the feature media because. Without it the image goes full height and I feel that is a waste of space. If you have a better way to control this please let me know!

```
.entry-header .featured-media {
    max-width: 680px;
    max-height: auto !important;
    margin: 0 auto;
  }
```

**Review by jnschrag — CHANGES_REQUESTED** [review]
A few more tweaks and I think this will be good to go. Thank you!

**Review by jnschrag — APPROVED** [review]
Nice job, @jdromero88! I don't know who prints websites, but 🤷‍♀️ they'll at least get a nice result if they do that here. :)

**Comment by jdromero88** [comment]
jajajajajajaja always cracking me up!!! Gracias Jacque!!!

**Pull request merged**